### ISSUE TITLE
Cannot stop podman machine after installing Podman Desktop

### ISSUE THREAD

**Issue by tnk4on** [containers/podman-desktop#3140]
### Bug description

I installed only the Podman CLI on Windows. In this case podman machine start/stop works fine.
If I then install Podman Desktop, `podman machine stop` does not work and the machine continues to start.
The machine does not stop when any of the following commands are executed.
`wsl --shutdown`,`wsl -t <machine name>`, `podman machine stop` 

### Operating system

Windows 11

### Installation Method

Installer from website/GitHub releases

### Version

1.1.0

### Steps to reproduce

Install Podman CLI only
```
PS C:\Users\user> podman machine init
PS C:\Users\user> podman machine start
PS C:\Users\user> podman machine ls
NAME                     VM TYPE     CREATED         LAST UP            CPUS        MEMORY      DISK SIZE
podman-machine-default*  wsl         31 seconds ago  Currently running  12          553.9MB     717.2MB
PS C:\Users\user> wsl -l -v
  NAME                      STATE           VERSION
* Ubuntu                    Stopped         2
  podman-machine-default    Running         2

PS C:\Users\user> podman machine stop
PS C:\Users\user> podman machine ls
NAME                     VM TYPE     CREATED             LAST UP        CPUS        MEMORY      DISK SIZE
podman-machine-default*  wsl         About a minute ago  7 seconds ago  0           0B          750.8MB
PS C:\Users\user> wsl -l -v
  NAME                      STATE           VERSION
* Ubuntu                    Stopped         2
  podman-machine-default    Stopped         2
```

Then install Podman Desktop. Stopped podman machine starts automatically.
Podman machine will not stop.
```
PS C:\Users\user> podman machine ls
NAME                     VM TYPE     CREATED        LAST UP            CPUS        MEMORY      DISK SIZE
podman-machine-default*  wsl         3 minutes ago  Currently running  12          569.7MB     721.4MB
PS C:\Users\user> wsl -l -v
  NAME                      STATE           VERSION
* Ubuntu                    Stopped         2
  podman-machine-default    Running         2

PS C:\Users\user> podman machine stop
Machine "podman-machine-default" stopped successfully
PS C:\Users\user> podman machine ls
NAME                     VM TYPE     CREATED        LAST UP            CPUS        MEMORY      DISK SIZE
podman-machine-default*  wsl         4 minutes ago  Currently running  12          581.9MB     721.4MB
PS C:\Users\user> wsl -l -v
  NAME                      STATE           VERSION
* Ubuntu                    Stopped         2
  podman-machine-default    Running         2

PS C:\Users\user> wsl --shutdown
PS C:\Users\user> wsl -l -v
  NAME                      STATE           VERSION
* Ubuntu                    Stopped         2
  podman-machine-default    Running         2

PS C:\Users\user> wsl -t podman-machine-default
The operation completed successfully.
PS C:\Users\user> wsl -l -v
  NAME                      STATE           VERSION
* Ubuntu                    Stopped         2
  podman-machine-default    Stopped         2
PS C:\Users\user> wsl -l -v
  NAME                      STATE           VERSION
* Ubuntu                    Stopped         2
  podman-machine-default    Running         2
```

**Comment by benoitf** [comment]
hello @tnk4on thanks for the report

Would it be possible for you to try a 'development version' of Podman Desktop at https://github.com/containers/podman-desktop/releases

There was a bug that was triggering automatic restart when going back to the dashboard. This bug is fixed in the development version so I would be interested to know if your problem is related and then fixed.

Thanks

**Comment by benoitf** [comment]
related issue: https://github.com/containers/podman-desktop/issues/2781

**Comment by tnk4on** [comment]
I tried the latest development version.
https://github.com/containers/podman-desktop/releases/tag/v0.0.202307071058-dd6cc8c

It works fine. No other problems are seen.

**Comment by benoitf** [comment]
thanks a lot @tnk4on 

I'll then close this issue as a duplicate of https://github.com/containers/podman-desktop/issues/2781

and problem will be solved in Podman Desktop 1.2.0 release

**Comment by benoitf** [comment]
dup of https://github.com/containers/podman-desktop/issues/2781
